Cargo.toml

```diff
@@ -21,3 +21,7 @@ members = ["."]
 
 [profile.release]
 lto = true
+strip = "symbols"
+panic = "abort"
+codegen-units = 1
+opt-level = 's'
```

Comment:
Why do we potentially choose size over performance? The code is supposed to be executed in the processing, so the slowness of the code will cost us money, and the customer costs latency. 

The same for `codegen-units`. From the docs:
> More code generation units allows more of a crate to be processed in parallel possibly reducing compile time, but may produce slower code.

Reply:
right, agree that `codegen-units = 1` is probably what we want

Reply:
Instead of trying to overload existing profile, we can perhaps create separate profile for `small`, `fast`, etc which can optimize for different target use cases.